src/lib/y2storage/storage_manager.rb

```diff
@@ -643,6 +643,10 @@ def retry_create_instance?(error, callbacks)
     class StorageLogger < ::Storage::Logger
       # rubocop:disable Metrics/ParameterLists
       def write(level, component, filename, line, function, content)
+        # libstorage pretent that it use same logging as y2_logger but y2_logger support also
```

Comment:
libstorage-ng does not pretend anything. If you look at Logger::write and tell me where the arguments to insert should some from I would be very happy.